scripts/buildkite/tpp-gpu.yml

```diff
@@ -20,7 +17,20 @@ steps:
     command: "${BUILDKITE_BUILD_CHECKOUT_PATH}/scripts/buildkite/check_llvm.sh"
   - wait
 
-  - label: "TPP-MLIR-clang"
+  - label: "TPP-MLIR-nvidia"
     command: "${SRUN} --partition=nv-v100 --time=0:30:00 -- \
               'KIND=Debug COMPILER=clang LINKER=lld GPU=1 SANITIZERS=1 CHECK=1 \
               $${BUILDKITE_BUILD_CHECKOUT_PATH}/scripts/buildkite/build_tpp.sh'"
+    env:
+      TPP_CLANGVER: "16.0.4"
+      TPP_GCCVER: "9.5.0" # Use an older version to match CUDA requirements
+      TPP_CUDAVER: "11.5"
+
+  - label: "TPP-MLIR-intel"
+    command: "${SRUN} -- \
+              'KIND=Debug COMPILER=clang LINKER=lld GPU=1 CHECK=1 \
+              $${BUILDKITE_BUILD_CHECKOUT_PATH}/scripts/buildkite/build_tpp.sh'"
+    env:
```

Comment:
I'd imagine the Intel version would need at least `TPP_CLANGVER`

Reply:
The PR is not yet ready for review. It does not even build and is mostly about getting SSH-based execution on target system up.

Reply:
I get it, just making some en passant comments. :)